src/test/java/uk/gov/hmcts/darts/event/service/impl/EventMappingServiceImplTest.java

```diff
@@ -231,6 +231,27 @@ void getEventMappingById(boolean hasEvents) {
 
     }
 
+    @Test
+    void deleteEventMappingById() {
+        EventHandlerEntity eventHandlerEntity = new EventHandlerEntity();
```

Comment:
could class variable eventHandlerEntity be re-used here - although they are slightly different?

Reply:
done